### DISCUSSION TITLE
Change captions/subtitle size font on player

### DISCUSSION THREAD

**Discussion by nitrocell** [sampotts/plyr#2671] [Ideas]
Hello,

Would it be possible to create a feature to change captions/subtitle formatting such as size and font on video player controls like JWplayer has natively or even VideoJS has some plugins to do so?

**Comment by belbernard** [comment]
I am also looking for a solution… Apparently this player does not react to CSS "cue::" formatting, unlike Gutenberg's default Wordpress video player.
Even chatGPT was unable to assist me! :-)

**Reply by belbernard** [reply]
In fact it does take ::cue CSS formatting contained in WebVTT files, which makes it possible to change colors, text size and font family.
